src/plugins/intel_gpu/src/plugin/plugin.cpp

```diff
@@ -651,7 +651,6 @@ std::vector<ov::PropertyName> Plugin::get_supported_properties() const {
         ov::PropertyName{ov::hint::dynamic_quantization_group_size.name(), PropertyMutability::RW},
         ov::PropertyName{ov::hint::activations_scale_factor.name(), PropertyMutability::RW},
         ov::PropertyName{ov::weights_path.name(), PropertyMutability::RW},
-        ov::PropertyName{ov::cache_encryption_callbacks.name(), PropertyMutability::RW},
```

Comment:
I think it's allows to be `WO`
Probably, for `CompiledModel` we can do the same - it can be allowed to set encryption callback for `CompiledModel` and call `export_model` then

Reply:
maybe we need to fix `hello_query_device ` ? 

BTW, potential place which need to be fixed
https://github.com/openvinotoolkit/openvino/blob/e50d72232c2e864248eefc74de422ceda1b627cd/src/inference/include/openvino/runtime/properties.hpp#L195-L197

Reply:
You're right, allowing WO to be recognized as mutable fixes the hello_query_device issue. But what's the point of exposing it? It's not needed for CompiledModel.

Reply:
> maybe we need to fix `hello_query_device ` ?
> 
> BTW, potential place which need to be fixed
> 
> https://github.com/openvinotoolkit/openvino/blob/e50d72232c2e864248eefc74de422ceda1b627cd/src/inference/include/openvino/runtime/properties.hpp#L195-L197

created a ticket for that: ref [CVS-161834](https://jira.devtools.intel.com/browse/CVS-161834)

Reply:
hello_query_device is one of the simplest samples, if that crashes, user code will also crash. It is quite common to iterate over supported_properties. Changes like that should only be introduced in a .0 release. My preference is not exposing this property, or not showing in SUPPORTED_PROPERTIES.

Reply:
> not showing in SUPPORTED_PROPERTIES

It should be shown here, otherwise does not know whether they can pass encryption callbacks for specific device (not all devices does take it into account)

Reply:
Changed it back to WO